examples/relay-angular-chat/src/app/app.component.ts

```diff
@@ -0,0 +1,82 @@
+import { Component } from '@angular/core';
+import { Waku, WakuMessage } from 'js-waku';
+import protons from "protons";
+
+const proto = protons(`
+message SimpleChatMessage {
+  uint64 timestamp = 1;
+  string text = 2;
+}
+`);
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
+})
+
+export class AppComponent {
+  
+  contentTopic: string = `/relay-angular-chat/1/chat/proto`;
+  title: string = 'relay-angular-chat';
+  // TODO: Probably better to create a Message interface and use that type here rather than any.
+  messages: any[] = [];
+  messageCount: number = 0;
+  // TODO: The other ignores are because of this. the Waku type isn't being picked up for some reason.
```

Comment:
Maybe play around with the published package see if it helps? Doing this angular exercise is meant to dig up bugs like this one.

Reply:
All of these have been resolved 🥳